DOCS-6035 API Notebook security (#70)
mulesoft/docs-exchange: modules/ROOT/pages/to-use-api-notebook.adoc

----
====

> == API Notebook security

Reviewer: ```suggestion
== API Notebook Security
```

API notebooks run plain JavaScript in the browser. To ensure sensitive data does not leak, API Notebook disables certain JavaScript functionality:

> 

Reviewer: You don't need to end the following list items with periods, because they are both fragments.

Reviewer 2: I'm a little embarrassed to have missed this, and to have missed the capitalization. Thank you very much for fixing them both!

> API notebooks run plain JavaScript in the browser. To ensure this does not leak sensitive data, API Notebook disables certain JavaScript functionality:

Reviewer 3: ```suggestion
API notebooks run plain JavaScript in the browser. To ensure sensitive data is not leaked, API Notebook disables certain JavaScript functionality:
```

Author: Done!
